### DISCUSSION TITLE
How can I use 'Free public server'?

### DISCUSSION THREAD

**Discussion by pgkmop** [rustdesk/rustdesk#5991] [Q&A]
Hello!
https://github.com/rustdesk/rustdesk/#free-public-servers
What's this? Is it the list of _free public server_ providers? Do I understand it right that it could be used as a replacement for the default rendezvous server rs-ny.rustdesk.com [216.128.135.246] ?
Can you be more specific then how can I use that servers? I mean what the settings are I should specify in the Rustdesk application to use any of that? Or do you just mean I can register there for _free_ and create my own Rustdesk server? I don't see such options there neither...

**Comment by jexjws** [comment]
I think "Free Public Servers" probably refers to the servers used by the free service rs-ny.rustdesk.com: when you use Rustdesk for free through rs-ny.rustdesk.com,   "the default rendezvous server"  are donated by these providers.

**Comment by rc621** [comment]
Has the free public server been shut down since today [31.01.2024] ?
I keep getting the error message "Failed to connect to rs-ny.rustdesk.com:21116"
it has always worked very well so far - congratulation and thanks for that free software.

**Reply by DMSnz** [reply]
https://github.com/rustdesk/rustdesk/discussions/7008
